src/index.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>To Do List</title>
+  </head>
+  <body>
+    <div class="container">
+      <p class="title">Today's To Do</p>
+      <div class="input-container">
+        <input class="add" placeholder="Add to your list..." />
+        <img src="./images/enter.jpg" alt="" />
+      </div>
+      <ul class="task-container">
+        <li>Test <img src="" alt="" /></li>
+      </ul>
+      <button class="remove">Clear all completed</button>
+    </div>
+    <img src="./images/icon.jpg" alt="" . />
```

Comment:
- [ ] I don't think this needs to be here 
![image](https://user-images.githubusercontent.com/62705947/236024607-ec99a8df-02ba-48a8-a9dc-af9616757eaa.png)